hrms/hr/doctype/appraisal_cycle/appraisal_cycle.js

```diff
@@ -3,6 +3,7 @@
 
 frappe.ui.form.on("Appraisal Cycle", {
 	refresh(frm) {
+		hrms.appraisal_utils.set_autocompletions_for_final_score_formula(frm)
```

Comment:
Let's call it `performance_utils`? Or we could write the function here itself since unlike payroll this is not being used in 3 different doctypes. So instead of creating a separate bundle, can be done here too.